tests/bootstrap

```diff
@@ -50,27 +50,16 @@ EOF
 
 source "${SCRIPT_DIR}/bootstrap_utils.sh"
 
-PARAMS=$(getopt -o "o:h" --name ${BASENAME} -- "$@")
-
-eval set -- "$PARAMS"
-unset PARAMS
-
-while true ; do
-    case $1 in
-        -o)
-            BUILDDIR=$2
-            shift 2
+while getopts "o:h" opt ; do
+    case ${opt} in
+        o)
+            BUILDDIR="${OPTARG}"
             ;;
-        -h)
+        h)
             usage
             exit
             ;;
-        --)
-            shift
-            break
-            ;;
         *)
```

Comment:
We are calling getopts in normal (non-silent) mode.
That means on an unknown option, we will get '?' in $opt.
If getopts sees -o without an argument, getopts will also put '?' in $opt.

Prefer to look for '?' in this case rather than *